README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/aws/aws-xray-java-agent.svg?branch=master)](https://travis-ci.com/aws/aws-xray-java-agent)
+[![Actions Status](https://github.com/aws/aws-xray-java-agent/workflows/Continuous%20Build/badge.svg)](https://github.com/aws/aws-xray-java-agent/actions)
```

Comment:
```suggestion
[![Build Status](https://github.com/aws/aws-xray-java-agent/workflows/Continuous%20Build/badge.svg)](https://github.com/aws/aws-xray-java-agent/actions)
```

Actions is the implementation detail of how we implement our build, think the later is still a more familiar term.